README.md

```diff
@@ -29,6 +29,8 @@ jobs:
 
 Available options:
 - publishToBinaries (default: false): enable the publication to binaries
+- publishJavadoc (default: false): enable the publication of the javadoc to https://javadocs.sonarsource.org/
+- javadocDestinationDirectory (required when publishJavadoc is true): define the subdir to use in https://javadocs.sonarsource.org/
```

Comment:
Cannot it be by default the name of the plugin/project?

Reply:
interesting feature, I will implement that

Reply:
done here: 261dd48112d0b2a2ac4ea9a0306c659e0b132df8